keras/layers/convolutional.py

```diff
@@ -434,83 +407,38 @@ class Convolution3D(Layer):
         `(samples, new_conv_dim1, new_conv_dim2, new_conv_dim3, nb_filter)` if dim_ordering='tf'.
         `new_conv_dim1`, `new_conv_dim2` and `new_conv_dim3` values might have changed due to padding.
     '''
-
     def __init__(self, nb_filter, kernel_dim1, kernel_dim2, kernel_dim3,
                  init='glorot_uniform', activation='linear', weights=None,
                  border_mode='valid', subsample=(1, 1, 1), dim_ordering=K.image_dim_ordering(),
                  W_regularizer=None, b_regularizer=None, activity_regularizer=None,
                  W_constraint=None, b_constraint=None,
                  bias=True, **kwargs):
-        if border_mode not in {'valid', 'same'}:
-            raise Exception('Invalid border mode for Convolution3D:', border_mode)
-        self.nb_filter = nb_filter
+        super(Convolution3D, self).__init__(nb_filter,
```

Comment:
Working hard to follow the indentation style but to be honest it's pretty tiring having to keep on tabbing and spacing things into position, and the lines easily run over 80 chars. I often work with multiple IDE windows side by side so 80 char limits help to avoid word-wrap/having to scroll.

How strict is this style? Is it a problem if some code is indented e.g.:

        super(Convolution3D, self).__init__(
            nb_filter,
            dimensions=(kernel_dim1, kernel_dim2, kernel_dim3),
            init=init,
            ...
            bias=bias,
            **kwargs
        )